PowerPointLabs/PowerPointLabs/ColorsLab/ColorsLabPaneWPF.xaml.cs

```diff
@@ -1265,57 +1237,17 @@ private void EmptyThemePanel()
         /// </summary>
         /// <param name="name"></param>
         /// <param name="color"></param>
-        private void SetThemeColorRectangle(string name, Color color)
+        private void SetThemeColorRectangle(int column, Color color)
         {
-            switch (name)
-            {
-                case "favoriteColorRectangleOne":
-                dataSource.ThemeColorOne = color;
-                break;
-                case "favoriteColorRectangleTwo":
-                dataSource.ThemeColorTwo = color;
-                break;
-                case "favoriteColorRectangleThree":
-                dataSource.ThemeColorThree = color;
-                break;
-                case "favoriteColorRectangleFour":
-                dataSource.ThemeColorFour = color;
-                break;
-                case "favoriteColorRectangleFive":
-                dataSource.ThemeColorFive = color;
-                break;
-                case "favoriteColorRectangleSix":
-                dataSource.ThemeColorSix = color;
-                break;
-                case "favoriteColorRectangleSeven":
-                dataSource.ThemeColorSeven = color;
-                break;
-                case "favoriteColorRectangleEight":
-                dataSource.ThemeColorEight = color;
-                break;
-                case "favoriteColorRectangleNine":
-                dataSource.ThemeColorNine = color;
-                break;
-                case "favoriteColorRectangleTen":
-                dataSource.ThemeColorTen = color;
-                break;
-                case "favoriteColorRectangleEleven":
-                dataSource.ThemeColorEleven = color;
-                break;
-                case "favoriteColorRectangleTwelve":
-                dataSource.ThemeColorTwelve = color;
-                break;
-                default:
-                break;
-            }
+            dataSource.SetFavoriteColor(column, color);
         }
 
         /// <summary>
         /// Save current panel as default theme color.
         /// </summary>
         private void SaveDefaultColorPaneThemeColors()
         {
-            dataSource.SaveThemeColorsInFile(_defaultThemeColorDirectory);
+            dataSource.SaveFavoriteColorsInFile(_defaultThemeColorDirectory);
```

Comment:
Default color pane theme colours are related to favourite colors now?

Reply:
@leeyh20 In the code it seems that they are mean the same thing. Did I misinterpret it?

Reply:
The change here is because I renamed the other function. Should I rename this as well in this PR?

Reply:
@leeyh20 @YuPeiHenry Yes, i believe they mean the same thing. When I was porting over i think I may have messed up some of the naming and it became inconsistent. Will be good if you can standardise this, Henry.

Reply:
Let's use `Favourite Colors` since that's the label on the pane